dashboard/app/jobs.go

```diff
@@ -666,6 +667,47 @@ func gatherCrashTitles(req *dashapi.JobDoneReq) []string {
 	return ret
 }
 
+// resetJobs is called by syz-ci to "unstart" previously started jobs.
+func resetJobs(c context.Context, req *dashapi.JobResetReq) error {
+	var jobs []*Job
+	keys, err := db.NewQuery("Job").
+		Filter("Finished=", time.Time{}).
+		Filter("Started>", time.Time{}).
+		GetAll(c, &jobs)
+	if err != nil {
+		return err
+	}
+	managerMap := map[string]bool{}
+	for _, name := range req.Managers {
+		managerMap[name] = true
+	}
+	for idx, job := range jobs {
+		if !managerMap[job.Manager] {
+			continue
+		}
+		jobKey := keys[idx]
+		tx := func(c context.Context) error {
+			job = new(Job)
+			if err := db.Get(c, jobKey, job); err != nil {
+				return fmt.Errorf("job %v: failed to get in tx: %v", jobKey, err)
+			}
+			if !job.Finished.IsZero() {
```

Comment:
!job.Finished.IsZero() is a double negation. It is hard to understand what is expected here.
job.IsFinished() or something similar is needed here.